src/Nethermind/Nethermind.Runner/Logging/NLogConfigurator.cs

```diff
@@ -44,7 +44,7 @@ public static class NLogConfigurator
                         {
                             foreach (Target? ruleTarget in rule.Targets)
                             {
-                                if (ruleTarget.Name == "seq")
+                                if (ruleTarget.Name == "seq" && rule.LoggerNamePattern == "*")
```

Comment:
why is this change needed?

Reply:
If the user create a rule to send a specific pattern to just seq. Then this will replace it's log level unintentially.